harness/testTypedArray.js

```diff
@@ -20,11 +20,22 @@ var typedArrayConstructors = [
   Uint8ClampedArray
 ];
 
+var numericTypedArrayConstructors = typedArrayConstructors.slice();
+
+if (typeof BigInt !== "undefined") {
+  typedArrayConstructors.push(BigInt64Array);
+  typedArrayConstructors.push(BigUint64Array);
+}
```

Comment:
this assumes by setting the constructor BigInt will also set the BigInt typed arrays.

Reply:
There is a strategy thing here, considering phenomenon the tests might all pass while `BigInt` is not implemented or used as a word.

At the point someone start implementing BigInt, just by setting any value to `BigInt` these tests will suddenly fail and not differentiate on what was already passing.

At some point, this might be an inheritance problem from the old TypeArray tests, not friendly to extensions, and I wrote them before I had much experience using the test generation tool.

In any case, they should not be failing as a whole.

Reply:
This would also require all the changed tests to flag BigInt in the feature flags, but it's weird to say they are also compatible if BigInt doesn't exist and we might be excluding all the respective typedArray tests if someone wants to filter out the BigInt feature.

I think it's time to clean up my mistake and generate individual tests for each typed array, otherwise we will have to deal with this in further extensions, if any.

Reply:
What about using `eval('1n')` to detect BigInt support?